modules/swagger-jaxrs2/src/main/java/io/swagger/v3/jaxrs2/DefaultParameterExtension.java

```diff
@@ -162,7 +164,7 @@ private boolean handleAdditionalAnnotation(List<Parameter> parameters, Annotatio
                 if (field != null) {
                     paramType = field.getType();
 
-                    for (final Annotation fieldAnnotation : field.annotations()) {
+                    for (final Annotation fieldAnnotation : annotations(field.getAllAnnotations())) {
```

Comment:
see also general review, we would keep as is to maintain support for jackson 2.8.x